.ci/validation.sh

```diff
@@ -133,6 +133,20 @@ no-violation-test-josm)
   if [[ ${RESULT} != 0 ]]; then false; fi
   ;;
 
+javac15)
+  files=($(grep -Rl --include='*.java' ': Compilable with Java15' \
+        src/test/resources-noncompilable || true))
+  if [[  ${#files[@]} -eq 0 ]]; then
+    echo "No Java15 files to process"
+  else
+      mkdir -p target
```

Comment:
`mkdir` and `echo` don't have the same indentation.

Reply:
Fixed.